packages/helpers/__tests__/product/isProductSimple.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { isProductSimple } from "@shopware-pwa/helpers";
```

Comment:
all helpers tests should cover more cases like:
- `isProductSimple()`
- `isProductSimple({product: null})`
- `isProductSimple({})`